### ISSUE TITLE
make: *** [m013330.mk:4: m013330.abyss2-contigs.fa] Error 2

### ISSUE THREAD

**Issue by Soleilcode** [adigenova/wengan#78]
Hi
I am using Wengen for short read files from Illumina and long read from pacbio CLR to assemble a human genome with the following command：
`/public1/data/heke/pacbio/wengan-v0.2-bin-Linux/wengan.pl -x pacraw -a A -s /public2/data/laixianxin/pacbio/ngs/cleandata/zhangqianqian_L1_353353.R1.clean.fastq.gz,/public2/data/laixianxin/pacbio/ngs/cleandata/zhangqianqian_L1_353353.R2.clean.fastq.gz -l /public1/data/heke/pacbio/fa/m013330.fasta.gz -p m013330 -t 20 -g 100`
 Then i got an errror in log:
gzip: stdout: Broken pipe
make: Warning: File 'm013330.mk' has modification time 275 s in the future
make: *** [m013330.mk:4: m013330.abyss2-contigs.fa] Error 2

I see error 2 in the existing issues, but his problem is not the same as mine, so I don't know how to solve this problem.

And my CLR data coverage is not very high, only around 15x. Is it necessary to adjust the parameters, such as reducing the values of the - d and - N parameters, to retain more contigs.

Could you give me some advice please?

**Comment by Soleilcode** [comment]
I checked the m013330.abyss2.err. It said:
Running with max stack size of 65536 KB: abyss-bloom-dbg -k96 -q3 --kc=3 -v -b20G -H4 -j20 /public2/data/laixianxin/pacbio/ngs/cleandata/zhangqianqian_L1_353353.R1.clean.fastq.gz /public2/data/laixianxin/pacbio/ngs/cleandata/zhangqianqian_L1_353353.R2.clean.fastq.gz
abyss-bloom-dbg: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by abyss-bloom-dbg)
abyss-bloom-dbg: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by abyss-bloom-dbg)
abyss-bloom-dbg: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by abyss-bloom-dbg)
make[1]: *** [/public1/data/heke/pacbio/wengan-v0.2-bin-Linux/bin/abyss-pe:538: m013330.abyss2-1.fa] Error 1
make[1]: *** Deleting file 'm013330.abyss2-1.fa'

But i used command: strings /lib64/libstdc++.so.6|grep GLIBCXX
Then i found there indeed a GLIBCXX_3.4.21.

strings /lib64/libstdc++.so.6|grep GLIBCXX
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_3.4.21
GLIBCXX_3.4.22
GLIBCXX_3.4.23
GLIBCXX_3.4.24
GLIBCXX_3.4.25
GLIBCXX_3.4.26
GLIBCXX_3.4.27
GLIBCXX_3.4.28
GLIBCXX_3.4.29
GLIBCXX_DEBUG_MESSAGE_LENGTH

I don't know how i can fix this issue.

**Comment by Soleilcode** [comment]
I have successfully solved this problem. There is only libstdc++.so.6.0.19 on the compute node. I asked the root administrator to copy libstdc++.so.6.0.29 in my shared directory on the login node to /lib64/ on the compute node. 
And I looked through the posts about assembly recommendations for low coverage data. My final parameters was "-M 1000-d 2-N 3-i 500,1000 ,2000,3000,4000,5000,6000,7000,8000,9000,10000,11000,12000,13000,14000,15000,20000" . Hope to get better result.
